tests/src/main/scala/sttp/tapir/tests/package.scala

```diff
@@ -315,6 +315,9 @@ package object tests {
       .out(stringBody)
       .name("Query with default")
 
+  val out_overridden_content_type_header: Endpoint[Unit, Unit, String, Any] =
```

Comment:
[minor, naming] this is in other places referred to as a `fixed` header - instead of `overriden`